tests/integration/introduction/test_introduction.py

```diff
@@ -0,0 +1,33 @@
+from tests.integration.integration_test_case import IntegrationTestCase
+from tests.integration.create_token import create_token
+
+
+class TestIntroduction(IntegrationTestCase):
+
+    def test_introduction_has_email_link(self):
```

Comment:
Yeah I guess you're right. I can remove it - I initially added it as a sanity check to make sure the survey I chose for the tests had an introduction page.